frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown"></TextBlock>
+        <div class="flex justify-between mt-4">
+            <span>Current global limit</span>
+            <span v-if="liquidationLimits"
+                ><Explain :text="liquidationLimits.maximumProtocolDebtDai.toFixed()"></Explain> -
+                <Explain :text="liquidationLimits.currentProtocolDebtDai.toFixed()"></Explain> -
```

Comment:
Please provide explanations to the numbers (please propose them if there are none so far). Eg
```
The amount of DAI needed to cover debt and fees of all active auctions. The internal name for this parameter is [dog.Dirt()](https://github.com/makerdao/dss/blob/fa4f6630afb0624d04a003e920b0d71a00331d98/src/dog.sol#L78)
```

More info can be found here https://github.com/sidestream-tech/unified-auctions-ui/issues/363#issuecomment-1234396013